manage-application-state-with-jotai-atoms/01-react-share-state-between-react-components-with-jotai-useatom.md

```diff
@@ -0,0 +1,27 @@
+## 01. Share State Between React Components with Jotai useAtom
+
+<Timestamp start="0:25" end="0:35">
```

Comment:
To keep things consistent, we should have it as `<TimeStamp` instead of `<Timestamp>`. That's how we have it in the rest of the notes.

Reply:
good catch 👍